### ISSUE TITLE
API for getting/setting values from range of cells

### ISSUE THREAD

**Issue by SathishRaju** [rowsncolumns/grid#146]
Is there a way to programmatically set and get values of range of cells? I looked at the **ref**, but it does not have any related methods.

**Comment by rmdort** [comment]
SpreadSheet state is driven by the user. So you control the state

https://rowsncolumns.app/docs/controlled-uncontrolled

**Comment by SathishRaju** [comment]
Yes, thats how am passing sheets data to the spreadsheet. But it will be nice if there is an interface for cell ranges. Currently the data has to be in the **CellConfig** format. And if i want to change the data or get data from a range of cells, then i need to write code to parse/update those values there.
Google sheets and excel provides an api to get/set values for a range of cells. 
If you can introduce additional methods through the **ref**, it would be great.

**Comment by rmdort** [comment]
To `set` state, you can use this dispatch method

https://rowsncolumns.app/docs/controlled#imperative-methods

To `get` state, we dont have an API, you will have to read it from your local state.
